test/acceptance/tests.py

```diff
@@ -389,64 +386,6 @@ def test_peer_assessment(self):
         self.do_peer_assessment()
 
 
-class PeerAssessmentTestStaffOverride(OpenAssessmentTest):
```

Comment:
I had thought that by leaving the third test (`full_workflow_staff_override`, which includes self and peer grading IIRC) we'd cover all the bases. I can leave these tests in though if you still think the difference in setup is worth it.